src/main/java/mil/dds/anet/database/PersonDao.java

```diff
@@ -457,4 +459,60 @@ public String clearCache() {
     return AnetConstants.USERCACHE_EMPTY_MESSAGE;
   }
 
+  @InTransaction
+  public int updatePersonHistory(Person p) {
+    final String personUuid = p.getUuid();
+    // Delete old history
+    final int numRows = getDbHandle()
+        .execute("DELETE FROM \"peoplePositions\"  WHERE \"personUuid\" = ?", personUuid);
+    // Add new history
+    for (final PersonPositionHistory history : p.getPreviousPositions()) {
+      updatePeoplePositions(history.getPositionUuid(), personUuid, history.getStartTime(),
+          history.getEndTime());
+    }
+    return numRows;
+  }
+
+  void updatePeoplePositions(final String positionUuid, final String personUuid,
+      final Instant startTime, final Instant endTime) {
+    if (endTime == null) {
+      // we have to make an exception here, as MSSQL has problems inserting a null datetime
+      getDbHandle()
+          .createUpdate("INSERT INTO \"peoplePositions\" "
+              + "(\"positionUuid\", \"personUuid\", \"createdAt\") "
+              + "VALUES (:positionUuid, :personUuid, :createdAt)")
+          .bind("positionUuid", positionUuid).bind("personUuid", personUuid)
+          .bind("createdAt", DaoUtils.asLocalDateTime(startTime)).execute();
+    } else {
+      getDbHandle()
+          .createUpdate("INSERT INTO \"peoplePositions\" "
+              + "(\"positionUuid\", \"personUuid\", \"createdAt\", \"endedAt\") "
+              + "VALUES (:positionUuid, :personUuid, :createdAt, :endedAt)")
+          .bind("positionUuid", positionUuid).bind("personUuid", personUuid)
+          .bind("createdAt", DaoUtils.asLocalDateTime(startTime))
+          .bind("endedAt", DaoUtils.asLocalDateTime(endTime)).execute();
+    }
+  }
+
+  public boolean hasPositionConflict(Person p) {
+    boolean opResult = false;
+    final String personUuid = p.getUuid();
+    List<PersonPositionHistory> previousPositions = p.getPreviousPositions();
+    for (PersonPositionHistory hist : previousPositions) {
+      final Instant startTime = hist.getStartTime();
+      final Instant endTime = hist.getEndTime();
+      final String positionUuid = hist.getPositionUuid();
+      final List<Map<String, Object>> rs = getDbHandle().createQuery(
+          "SELECT COUNT(*)  FROM \"peoplePositions\" AS count WHERE (( \"createdAt\" BETWEEN :startTime AND :endTime ) OR"
+              + "( \"endedAt\" BETWEEN :startTime AND :endTime) OR (:startTime BETWEEN \"createdAt\" AND \"endedAt\") OR ( :endTime BETWEEN \"createdAt\" AND \"endedAt\" ) ) AND \"positionUuid\" = :posUuid AND \"personUuid\" != :personUuid ")
+          .bind("startTime", startTime).bind("endTime", endTime).bind("posUuid", positionUuid)
```

Comment:
`startTime` and `endTime` need to be converted to be used as parameters: `DaoUtils.asLocalDateTime(…)`